clients/drcachesim/tracer/physaddr.cpp

```diff
@@ -130,35 +140,41 @@ physaddr_t::virtual2physical(addr_t virt, OUT addr_t *phys, OUT bool *from_cache
         // XXX i#4014: Provide a similar option that doesn't flush and just checks
         // whether mappings have changed?
         use_cache = false;
-        last_vpage_ = PAGE_INVALID;
-        hashtable_clear(&v2p_);
+        memset(last_vpage_, static_cast<char>(PAGE_INVALID), sizeof(last_vpage_));
```

Comment:
So we don't reset `last_ppage_` because it won't be used if we reset `last_vpage`? Perhaps worth a comment.

Reply:
Yes.  Done.